GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/UIElementDropBehavior.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Interactivity;
+using System.Windows.Media;
+using System.Windows.Media.Effects;
+
+namespace GalaxyZooTouchTable.Behaviors
+{
+    public class UIElementDropBehavior : Behavior<UIElement>
+    {
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+
+            AssociatedObject.TouchEnter += AssociatedObject_TouchEnter;
+            AssociatedObject.TouchLeave += AssociatedObject_TouchLeave;
+        }
+
+        private void AssociatedObject_TouchLeave(object sender, TouchEventArgs e)
+        {
+            Border element = sender as Border;
+            element.Background = Brushes.Transparent;
+            element.Effect = null;
+
+            Border childBorder = element.Child as Border;
+            childBorder.Background = Brushes.Transparent;
+            childBorder.Effect = null;
+        }
+
+        private void AssociatedObject_TouchEnter(object sender, TouchEventArgs e)
+        {
+            Border element = sender as Border;
+            LinearGradientBrush brush = new LinearGradientBrush();
+            brush.StartPoint = new Point(0, 0.5);
+            brush.EndPoint = new Point(1, 0.5);
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#727272"), 0));
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#202020"), 1));
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#000000"), 1));
+            element.Background = brush;
+
+            Border childBorder = element.Child as Border;
+            childBorder.Background = (SolidColorBrush)(new BrushConverter().ConvertFrom("#4DE5FF4D"));
+            DropShadowEffect childEffect = new DropShadowEffect();
+            childEffect.BlurRadius = 20;
+            childEffect.Color = Color.FromArgb(120,255,255,255);
```

Comment:
I think it would probably be best if you stuck to one way of defining a color. Either the `ColorConverter` used above or the `Color` you use here. Personally, I think the `Color` lib looks better

Reply:
`Color.FromArgb` will still be necessary for colors that need an opacity or aren't defined in the `Colors.COLOR_NAME` namespace.

Reply:
Also, some of these properties require the `SolidColorBrush` namespace, while others want the `Color` namespace, which is why there's a difference.

Reply:
I see. That's an unfortunate code pattern, but looks like it's what you have to do. 👍